app/views/tag_groups/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :title_bar do %>
+  <% if policy(TagGroup).new? %>
+    <%= link_to t('new tag_group'), new_tag_group_path, class: 'btn btn-primary btn-sm navbar-btn' %>
+  <% end %>
+  <%= link_to t('back'), root_path, class: "btn btn-primary btn-sm navbar-btn" %>
+<% end %>
+
+<div id="projects" class="projects-list">
+  <div class="row item">
+    <div class="col-md-12">
+        <section>
+          <h1>Tag Groups</h1>
+
+          <table class="table tag-groups">
+            <thead>
+              <tr>
+                <th>Color</th>
+                <th>Name</th>
+                <th>Description</th>
+                <th>Actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              <% @tag_groups.each do |tag_group| -%>
+                <tr>
+                  <td>
+                    <div class="color_container">
+                      <div class="color" style="background-color: <%= tag_group.bg_color %>;">
+                      </div>
+                    </div>
+                  </td>
+                  <td><%= tag_group.name %></td>
+                  <td><%= tag_group.description %></td>
+                  <td>
+                    <%= link_to 'Edit', edit_tag_group_path(tag_group) %>
+                    <%= link_to "Delete", tag_group, :method => :delete, :data => {:confirm => 'Are you sure?'} %>
```

Comment:
Try to use the new hash format, and you need to put the 'Are you sure?' into `I18n` to translate this phrase to others languages.